app/src/main/res/layout/topic_play_story_summary.xml

```diff
@@ -19,84 +19,125 @@
+      android:orientation="vertical">
+
+      <LinearLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:layout_marginStart="16dp"
+        android:layout_marginTop="24dp"
+        android:layout_marginEnd="20dp"
+        android:layout_marginBottom="24dp"
+        android:gravity="center_vertical"
+        android:orientation="horizontal">
+
+        <LinearLayout
+          android:layout_width="wrap_content"
+          android:layout_height="wrap_content"
+          android:layout_gravity="center"
+          android:gravity="center"
+          android:orientation="vertical">
+
+          <FrameLayout
+            android:id="@+id/story_progress_container"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content">
+
+            <org.oppia.app.customview.SegmentedCircularProgressView
+              android:id="@+id/story_progress_view"
+              android:layout_width="48dp"
+              android:layout_height="48dp" />
+
+            <TextView
+              android:id="@+id/story_progress_percentage"
+              android:layout_width="wrap_content"
+              android:layout_height="wrap_content"
+              android:layout_gravity="center"
+              android:fontFamily="sans-serif-medium"
+              android:text="@{@string/topic_story_progress_percentage(storyPercentage)}"
+              android:textColor="@color/oppiaPrimaryText"
+              android:textSize="14sp"
+              android:visibility="@{storyPercentage != 0 ? View.VISIBLE : View.GONE}" />
+          </FrameLayout>
+
+          <TextView
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:layout_gravity="center_horizontal"
+            android:layout_marginTop="4dp"
+            android:fontFamily="sans-serif"
+            android:text="@{storyPercentage != 100? @string/status_in_progress : @string/status_completed}"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="10sp"
+            android:visibility="@{storyPercentage != 0 ? View.VISIBLE : View.GONE}" />
+        </LinearLayout>
+
+        <LinearLayout
+          android:layout_width="match_parent"
+          android:layout_height="wrap_content"
+          android:layout_marginStart="20dp"
+          android:gravity="center_vertical"
+          android:orientation="vertical">
+
+          <TextView
+            android:id="@+id/story_name_text_view"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            android:fontFamily="sans-serif-medium"
+            android:onClick="@{viewModel::clickOnStorySummaryTitle}"
+            android:text="@{viewModel.storySummary.storyName}"
+            android:textColor="@color/oppiaPrimaryText"
+            android:textSize="20sp" />
+
+          <TextView
```

Comment:
The spacing between chapter_count textview and the dotted line (dashed) seems 7dp in mock even if its increased it should be 8dp is't?

Reply:
As mentioned earlier, the spacing between dashed line and chaptercount is dynamic and depends on the content in the card.

Reply:
I mentioned about the parent layout bottom spacing..

Reply:
It was not dynamic the parent Linear layout  of this view was having a 24dp bottom margin, which was not as per mock... It seems she updated mock now so you may follow the new mock updates.